.github/CODEOWNERS

```diff
@@ -206,7 +206,7 @@ WORKFLOW.md @torkelo
 /devenv/docker/ha_test/ @grafana/backend-platform
 /devenv/docker/loadtest/ @grafana/backend-platform
 /devenv/docker/rpmtest/ @grafana/backend-platform
-/devenv/jsonnet/ @grafana/grafana-edge-squad
+/devenv/jsonnet/ @grafana/dataviz-squad
```

Comment:
Not sure if dataviz should keep this vs. hand off to someone else (i.e. maybe to Todd in @grafana/multitenancy-squad)?